openraft/src/raft_types.rs

```diff
@@ -138,7 +137,8 @@ impl Display for SnapshotSegmentId {
 }
 
 // An update action with option to update with some value or just leave it as is.
-#[derive(Debug, Clone, PartialOrd, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialOrd, PartialEq, Eq)]
+#[cfg_attr(feature = "serde_impl", derive(serde::Deserialize, serde::Serialize))]
 pub enum Update<T> {
     Update(T),
     AsIs,
```

Comment:
`Update<T>` does not need to be serializable.
It must have been added by accident. :(